airbyte-integrations/connectors/source-waiteraid/source_waiteraid/waiteraid.yaml

```diff
@@ -0,0 +1,48 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: []
+  requester:
+    url_base: "https://app.waiteraid.com"
+    http_method: "POST"
+    request_options_provider:
+      request_parameters:
+        date: "{{ config['start_date'] }}"
+        auth_hash: "{{ config['auth_hash'] }}"
+        restid: "{{ config['restid'] }}"
+  stream_slicer:
+    type: "DatetimeStreamSlicer"
+    start_datetime:
+      datetime: "{{ config['start_date'] }}"
+      datetime_format: "%Y-%m-%d"
+    end_datetime:
+      datetime: "{{ now_utc() }}"
+      datetime_format: "%Y-%m-%d %H:%M:%S.%f"
+    step: "1d"
+    datetime_format: "%Y-%m-%d"
+    cursor_field: "{{ options['stream_cursor_field'] }}"
+  retriever:
+    record_selector:
+      $ref: "*ref(definitions.selector)"
+    paginator:
+      type: NoPagination
```

Comment:
It will limit retrieving data from only the first page of the service, is what you expected here?

Reply:
Oh, no that is not what I expect here. What would be the correct option for getting all pages?

Reply:
You can create paginator, but is not clear from API how it does.
Can you share the metadata from the API response for booking endpoints? (remove the data fields)

Reply:
I am not sure what exactly you need or how to get it. I asked the company behind the API for how optain metadata but didn't get a response yet. The data that you get is how they describe it on their website: https://app.waiteraid.com/api-docs/index.html#api_get_bookings